lib/availability_status_listener.rb

```diff
@@ -39,6 +40,11 @@ def process_event(event)
     Rails.logger.info("Kafka message #{event.message} received with payload: #{event.payload}")
     return unless event.message == EVENT_AVAILABILITY_STATUS
 
+    if (REQUIRED_HEADERS - event.headers.keys).any?
+      Rails.logger.error("Kafka message #{event.message} missing required header(s): #{(REQUIRED_HEADERS - event.headers.keys).join(',')}; returning.")
```

Comment:
Should we say:  ".. missing required header(s) #{REQUIRED_HEADERS}: found: #{event.headers.keys.join(',')}; returning"

So the `REQUIRED_HEADERS` are included in the log message?

Reply:
I like that idea - that way we can see the differences in formats they would have without having to debug in.